main.py

```diff
@@ -349,4 +358,4 @@ def wait_until_visible(driver, xpath=None, class_name=None, duration=10000, freq
     run(driver=driver, shoe_type=shoe_type, username=args.username, password=args.password, url=args.url, shoe_size=args.shoe_size,
         login_time=args.login_time, release_time=args.release_time, page_load_timeout=args.page_load_timeout,
         screenshot_path=args.screenshot_path, html_path=args.html_path, select_payment=args.select_payment,
-        purchase=args.purchase, num_retries=args.num_retries, dont_quit=args.dont_quit)
+        purchase=args.purchase, num_retries=args.num_retries, dont_quit=args.dont_quit, cvv = args.cvv)
```

Comment:
This should be registered with the parser using `add_argument`. Also needs to be added to the README.